packages/engine-core/src/data-proxy/utils/request.ts

```diff
@@ -102,7 +102,10 @@ async function nodeFetch(url: string, options: RequestOptions = {}): Promise<Req
     // we execute the https request and build a fetch response out of it
     const request = https.request(url, httpsOptions, (response) => {
       // eslint-disable-next-line prettier/prettier
```

Comment:
```suggestion
      // eslint-disable-next-line prettier/prettier
```

since you'd already reformatted it